runtime/web_platform_apis.md

```diff
@@ -1,12 +1,13 @@
 # Using Web Platform APIs
 
-Deno aims to use web platform APIs (like `fetch`) instead of inventing a new
-proprietary API where it makes sense. These APIs generally follow the
-specifications and should match the implementation in Chrome and Firefox. In
-some cases it makes sense to deviate from the spec slightly, because of the
-different security model Deno has.
+One way Deno simplifies web and cloud development is by using web platform APIs
+(like `fetch`) over proprietary APIs. This means If you've ever built for the
+browser, you're likely already familiar with Deno, and if you're learning Deno,
+you're also investing in your knowledge of the web.
 
-Here is a list of web platform APIs Deno implements:
+## Supported APIs
+
+Here's a list of supported web platform APIs Deno implements:
```

Comment:
Maybe mention that it's not a complete list and explain how one can use MDN to check if API is available in Deno?